header/Text/TextReaderFactory.h

```diff
@@ -5,9 +5,6 @@
 #ifndef GPU_KARP_RABIN_TEXTREADERFACTORY_H
 #define GPU_KARP_RABIN_TEXTREADERFACTORY_H
 
-#include <fstream>
```

Comment:
Maybe lack of this particular library is causing strange problem with reading only the last line of the input file.

Reply:
This header is included in TextFileReader.h file. It should be sufficient as TextFileReader is the only class that realy uses the file streams.

Reply:
I missed that, sorry. Ignore this comment.